### PR TITLE
[v6r18] Fixes when getting elasticDB parameters

### PR DESCRIPTION
**Pull request by fstagni**
And several other things for certification tests

### CODE REVIEW

**Comment by fstagni** [comment]
Approve this easy PR?

File: ConfigurationSystem/private/ConfigurationClient.py [hunk 1]
```diff
@@ -1,13 +1,15 @@
-from DIRAC.ConfigurationSystem.Client.Helpers.Path import cfgPath
-__RCSID__ = "$Id$"
 
-import types
 import os
+import types
```
Comment (chaen) [inline ConfigurationSystem/private/ConfigurationClient.py]: Why do you need types still ?

Comment (fstagni) [inline ConfigurationSystem/private/ConfigurationClient.py]: Removed

File: Core/scripts/dirac-externals-requirements.py [hunk 2]
```diff
@@ -69,14 +73,11 @@ def setInstallType( val ):
 
 gLogger.notice( "Requesting installation of %s" % ", ".join( [ "%s%s" % ( reqName, reqDict[ reqName ][0] ) for reqName in reqDict ] ) )
 
-from pip.index import PackageFinder
-from pip.req import InstallRequirement, RequirementSet
-from pip.locations import build_prefix, src_prefix
-
 requirement_set = RequirementSet(
-    build_dir = build_prefix,
+    build_dir = user_dir,
```
Comment (chaen) [inline Core/scripts/dirac-externals-requirements.py]: Does that mean that pip will perform the installation in the user space ?

Comment (fstagni) [inline Core/scripts/dirac-externals-requirements.py]: Yep

Comment (chaen) [inline Core/scripts/dirac-externals-requirements.py]: But this is not what you want I suppose. You rather want all the externals at the same place, and available for all users no ?

Comment (fstagni) [inline Core/scripts/dirac-externals-requirements.py]: This script is used in the special case when /RequiredExternals/ is found in releases.cfg. To be honest, I don't know by who (not by LHCb) and IIUC it's for some user installation, probably by pilots. It's an alternative way to have a package on the worker node, where only the user space is usable.

Comment (fstagni) [inline Core/scripts/dirac-externals-requirements.py]: Approve now?

Comment (chaen) [inline Core/scripts/dirac-externals-requirements.py]: Yes ! fine with me, too bad for the others if side effect :D

**Comment by chaen** [comment]
Approved

**Pull request merged**